test/integration/rt_channel_test.exs

```diff
@@ -470,4 +610,23 @@ defmodule Realtime.Integration.RtChannelTest do
     {:ok, socket} = WebsocketClient.connect(self(), @uri, @serializer, [{"x-api-key", token}])
     socket
   end
+
+  def rls_context(context) do
+    [tenant] = Repo.all(Tenant)
+
+    {:ok, db_conn} = Connect.lookup_or_start_connection(tenant.external_id)
+
+    clean_table(db_conn, "realtime", "presences")
+    clean_table(db_conn, "realtime", "broadcasts")
+    clean_table(db_conn, "realtime", "channels")
+    channel = channel_fixture(tenant)
+
+    if Map.has_key?(context, :policies) do
+      create_rls_policies(db_conn, context.policies, channel)
+    end
```

Comment:
this is of course totally fine as is, for completeness a common idiom is:

```suggestion
    if policies = context[:policies] do
      create_rls_policies(db_conn, policies, channel)
    end
```

Reply:
looks way better will apply